src/registrar/management/commands/generate_test_transition_domains.py

```diff
@@ -0,0 +1,60 @@
+"""Data migration: Generate fake transition domains, replacing existing ones."""
+
+import logging
+
+from django.core.management import BaseCommand
+from registrar.models import TransitionDomain, Domain
+
+logger = logging.getLogger(__name__)
+
+
+class Command(BaseCommand):
+    help = "Generate test transition domains from existing domains"
+
+    # Generates test transition domains for testing send_domain_invitations script.
+    # Running this script removes all existing transition domains, so use with caution.
+    # Transition domains are created with email addresses provided as command line
+    # argument. Email addresses for testing are passed as comma delimited list of
+    # email addresses, and are required to be provided.  Email addresses from the list
+    # are assigned to transition domains at time of creation.
+
+    def add_arguments(self, parser):
+        """Add command line arguments."""
+        parser.add_argument(
+            "-e",
+            "--emails",
+            required=True,
+            dest="emails",
+            help="Comma-delimited list of email addresses to be used for testing",
+        )
+
+    def handle(self, **options):
+        """Delete existing TransitionDomains.  Generate test ones."""
+
+        # split options[emails] into an array of test emails
+        test_emails = options["emails"].split(",")
```

Comment:
(Nice to have but optional) I'd add some error handling before this if options["emails"] is None, so we can throw a custom ValueError or something of that nature - "Parameter emails must be supplied" for example

Reply:
Thank you.  I considered this, but used required=True on the parser.add_argument, which essentially does the same thing.